src/test/java/org/broadinstitute/hellbender/tools/gvs/ingest/VetFieldEnumUnitTest.java

```diff
@@ -68,6 +79,29 @@ public void testAlleleSpecific() {
 
     }
 
+    @Test
+    public void testAlleleSpecificTwoAlts() {
+        // just focusing on the difference for AS annotations for non 0/x genotypes
+
+        VariantContextBuilder builderA =
+                new VariantContextBuilder("a","1",10329,10329,
+                        Arrays.asList(Allele.REF_C,Allele.ALT_A,Allele.ALT_T, Allele.NON_REF_ALLELE));
+
+
+        Genotype g = new GenotypeBuilder(SAMPLE_1)
+                .alleles(Arrays.asList(Allele.ALT_A, Allele.ALT_T))
+                .make();
+
+        builderA.attribute(AS_RAW_MAP_QUAL_RANK_SUM_KEY,"|-0.2,1|-2.5,1")
```

Comment:
I don't understand where these values are coming from.  There's only one sample, so there should only be one value, but that sample is heterozygous non-reference, so there should be no values since there's no ref.

Reply:
These are just made up to ensure that we **don't** return them (ie we're just defining the input data here, say from HC where we sometimes see this for GT 1/2).  The test below asserts this is not the value returned